src/webui/www/public/download.html

```diff
@@ -5,15 +5,64 @@
 	<title>QBT_TR(Add Torrent Link)QBT_TR</title>
 	<link rel="stylesheet" href="css/style.css" type="text/css" />
 	<script type="text/javascript" src="scripts/mootools-1.2-core-yc.js" charset="utf-8"></script>
-	<script type="text/javascript" src="scripts/download.js" charset="utf-8"></script>
 </head>
 <body>
+<iframe id="download_frame" name="download_frame" class="invisible" src="javascript:false;"></iframe>
+<form action="command/download" enctype="multipart/form-data" method="post" id="downloadForm" style="text-align: center;" target="download_frame">
 <center>
 	<br/>
 	<h2 class="vcenter">QBT_TR(Download Torrents from their URLs or Magnet links)QBT_TR</h2>
-	<textarea id="urls" rows="10"></textarea>
+	<textarea id="urls" rows="10" name="urls"></textarea>
 <p>QBT_TR(Only one link per line)QBT_TR</p>
-<input type="button" value="QBT_TR(Download)QBT_TR" id="downButton"/>
-	</center>
+<fieldset class="settings" style="border: 0; text-align: left;">
+<div class="formRow" style="margin-top: 6px;">
+	<label for="savepath" class="leftLabelLarge">QBT_TR(Save files to location:)QBT_TR</label>
```

Comment:
I think you need to add translatable strings `QBT_TR(XXXX)QBT_TR` to this file:
https://github.com/qbittorrent/qBittorrent/blob/master/src/webui/extra_translations.h